### PR TITLE
Revert "feat(flags): script arguments come after '--' (#3621)"

### PR DESCRIPTION
**Pull request by ry**
Due to complaints about ergonomics and because it breaks shebang on
linux.

This reverts commit 2d5457df15d8c4a81362bb2d185b5c6013faa1d8.

### CODE REVIEW

**Comment by nayeemrmn** [comment]
Shebangs work just as well on GNU coreutils 8.30+, for before that there is no way of setting interpreter flags on Linux. In other words this didn't break anything any more than the need for a permission flag does -- Linux shebangs are broken (except for GNU coreutils 8.30+) either way.

The following applies to all platforms: The other ergonomic problem is described here https://github.com/denoland/deno/pull/3621#issuecomment-573503192 and a solution is given here https://github.com/denoland/deno/pull/3621#issuecomment-574855942 but that solution depends on the unfixed bug reported in #3011, which was wrongly closed.

...

In other words, +1 for reverting and reopening #3011!
